modules/ztp-configuring-kernel-arguments-for-discovery-iso.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
```

Comment:
:_content-type: PROCEDURE

Reply:
> @rohennes noticed that both of your assemblies are missing the content_type metadata. Please add it and I will get you merged up. Thank you.

Thanks @mjpytlak - updated the PR accordingly